omod/src/main/webapp/pages/myaccount/changeDefaults.gsp

```diff
@@ -8,10 +8,41 @@
         primaryLocaleOptions.add([label: it.displayName, value: it.toString()])
     }
 %>
+<script type="text/javascript">
 
+  var value;
+  
+  jq(document).ready(function() {
```

Comment:
The page is already written in angular, this is should be an easy addition via angular rather than adding all this new code.